barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_format.test.cpp

```diff
@@ -64,6 +64,34 @@ TEST_F(AcirFormatTests, TestASingleConstraintNoPubInputs)
         .poly_triple_constraints = { constraint },
         .quad_constraints = {},
         .block_constraints = {},
+        .original_opcode_indices =
```

Comment:
I've had to add logic to create the original opcode indices in all the tests that use the acir format. I guess that I could extract this logic to a test utility function that generates mocked original opcode indices backwards, from the constraints. 
What would be the go-to approach here? if the function to generate the original opcode indices is a good way to go, where should such a test utility function live?

Reply:
Going with a separate mock module to use in tests